staff/templates/instafeed.html

```diff
@@ -0,0 +1,25 @@
+
+<div id="mui-panel" class="rightpanelbody-hidden">
+      <h2 style="text-align: center; padding-bottom: 10px; font-size: 35px; color: #474747;">Instagram Feed</h2><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.2" />
+      <div id="instafeed">
+        <!-- INSTAFEED STUFF DOWN HERE !!!DO NOT CODE DOWN BELOW!!! -->
+      </div>
+</div>
+
+<div class="img-display-container" onclick="this.style.display='none'" style="position: fixed; width: 100%; height: 100%; top: 62px; background: rgba(0, 0, 0, 0.6); z-index: 1; text-align: center; vertical-align: middle">
+  <span onclick="this.parentElement.style.display='none'" class="closebtn">&times;</span>
+    <img id="expandedImg" style="z-index: 3; overflow: visible; display: inline-block; border-radius: 45px 45px 45px 45px; padding-top: 10px">
+    <div id="imgtext" style="position: fixed; bottom: 0; width: 100%; height: 20%; overflow: scroll; text-align: left"></div>
+</div>
+
+<script>
+    function importImage(imgs) {
+      var expandImg = document.getElementById("expandedImg");
+      var imgText = document.getElementById("imgtext");
+      var $bodyEl = $('body');
+      expandImg.src = imgs.src;
+      imgText.innerHTML = imgs.alt;
+      expandImg.parentElement.style.display = "block";
+      main_body.addClass('mui-scroll-lock');
```

Comment:
This code is broken, there's no `main_body`. I added #37 and #38 to fix up this stuff.

Reply:
Ok I'll resolve this convo and we'll refer to the issues.